client/pages/layout/layout.nav.controller.js

```diff
@@ -1,20 +1,18 @@
-angular.module('probedock.layout').controller('LayoutNavCtrl', function(api, appSettings, orgs, profile, $rootScope, $scope, $state) {
+angular.module('probedock.layout').controller('LayoutNavCtrl', function(api, appSettings, orgs, profile, $rootScope, $scope, states) {
 
+  orgs.forwardData($scope);
+  profile.forwardData($scope);
   appSettings.forwardSettings($scope);
 
-  var state = $state.current;
-  $rootScope.$on('$stateChangeSuccess', function(event, toState, toStateParams) {
-    state = toState;
-    $scope.orgName = toStateParams.orgName;
+  var currentStateName = null;
+  states.onStateChangeSuccess($scope, true, function(state, params, resolves) {
+    currentStateName = state.name;
   });
 
   $scope.baseStateIs = function() {
```

Comment:
We should take some time a day to think about that:

```
$scope.baseStateIs = function() {
  console.log('here');
  ...
}
```
Accessing the page: /

In the console:
`here` appears -> 92 times